lib/client.js

```diff
@@ -28,10 +28,11 @@ class Client extends Emitter {
       debug({msg}, 'Received message from jambonz');
       switch (msg.type) {
         case 'session:new':
-          this._onSessionNew(ws, path, msg);
-          break;
         case 'session:adulting':
-          this._onSessionAdulting(ws, path, msg);
+          this._onSessionEvent(ws, path, msg);
```

Comment:
this doesnt make sense to me, as it changes the 'adulting' behavior

Reply:
No, they are the same as before, both session:new or session:adulting will emit session:new